content/applications/sales/sales/send_quotations/get_paid_to_validate.rst

```diff
@@ -1,53 +1,68 @@
-:doc:`/applications/finance/payment_acquirers/paypal`,
-:doc:`/applications/finance/payment_acquirers/authorize`, and others in the
-:doc:`/applications/finance/payment_acquirers` documentation.
+After checking the box beside the :guilabel:`Online Payment` feature, a link to the
+:guilabel:`Payment Acquirers` appears beneath it. Click that link to reveal the entire
+:guilabel:`Payment Acquirers` page.
+
+.. image:: get_paid_to_validate/payment-acquirers.png
+   :align: center
+   :alt: Payment acquirers page in the Odoo Sales application.
+
+On the :guilabel:`Payment Acquirers` page, payment acquirers can be selected and configured. Before
+creating (or modifying) a payment acquirer, be sure to review the documentation related to the ways
+payment acquirers can be used with Odoo, such as:
+
+- :doc:`/applications/finance/payment_acquirers/paypal`
+- :doc:`/applications/finance/payment_acquirers/authorize`
+- :doc:`/applications/finance/payment_acquirers`
 
 .. note::
-   If you are familiar with this documentation: :doc:`quote_template`; you
-   can activate or not the **Online Payment** feature for each template you use, under their
-   confirmation tab.
+   On quotation templates, the :guilabel:`Online Payment` feature is located under the
+   :guilabel:`Confirmation` tab.
+
+   .. image:: get_paid_to_validate/confirmation-tab.png
+      :align: center
+      :alt: Confirmation tab with the online payment feature.
+
+   On standard quotations, the :guilabel:`Online Payment` feature is located under the
+   :guilabel:`Other Info` tab.
+
+   .. image:: get_paid_to_validate/other-info.png
+      :align: center
+      :alt: Other Info tab with the online payment feature.
 
 Register a payment
 ==================
 
-After opening quotations from their received email, your customers will have different
-possibilities to make their online payments. For example:
+After opening quotations from the email they receive, customers are presented with different options
+to make online payments, in the :guilabel:`Pay with` section.
 
-.. image:: get_paid_to_validate/get_paid_2.png
+.. image:: get_paid_to_validate/validate-order.png
    :align: center
-   :class: img-thumbnail
-   :alt: How to register a payment on Odoo Sales?
+   :alt: How to register a payment on Odoo Sales.
 
 .. tip::
-   By clicking on the **Customer Preview** button, you will be able to see what your customers
-   will have to choose when it comes to payment.
+   Clicking the :guilabel:`Customer Preview` smart button on the quotation form provides a preview
+   of the quotation as the customer sees it, along with the various online payment options they can
+   choose from.
 
 .. seealso::
-   - :doc:`quote_template`
+   - :doc:`/applications/sales/sales/send_quotations/quote_template`
    - :doc:`/applications/finance/payment_acquirers`
    - :doc:`/applications/finance/payment_acquirers/paypal`
    - :doc:`/applications/finance/payment_acquirers/authorize`
```

Comment:
I feel we can remove these since they were stated previously, however, it doesnt hurt to leave them– up to you :)
```suggestion
```